test/test_torch.py

```diff
@@ -6109,6 +6109,14 @@ def test_scalar_check(self, device):
         self.assertEqual((1,), torch.erf(one_d).shape)
         self.assertEqual((1,), torch.erfc(one_d).shape)
 
+        # clamp
+        self.assertEqual((), torch.clamp(zero_d, min=0, max=1).shape)
```

Comment:
nit: the fact that these are `int`s is not a problem?

Reply:
nope, the function handles them correctly.